config/prod.exs

```diff
@@ -18,9 +18,9 @@ config :orbit, Oban,
      timezone: "America/New_York",
      crontab: [
        # the top of every hour, any day, any month, any day of the week
-       {"0 * * * *", Orbit.Import.PersonnelWorker},
+       {"*/5 * * * *", Orbit.Import.PersonnelWorker},
        # the bottom of every hour, any day, any month, any day of the week
```

Comment:
these comment descriptions are no longer accurate (that's fine, you don't need to update comments for temporary stuff like this)

Was this change just so you don't have to wait an hour to test it?

Reply:
Yep!